draft-ietf-httpbis-digest-headers.md

```diff
@@ -479,20 +486,20 @@ The entries in {{iana-hash-algorithm-table}} are registered by this document.
 ## HTTP Messages Are Not Protected In Full {#sec-limitations}
 
 This document specifies a data integrity mechanism that protects HTTP
-`representation data` or content, but not HTTP header and trailer fields, from
+*representation data* or content, but not HTTP header and trailer fields, from
 certain kinds of corruption.
 
-Digest fields are not intended to be a general protection against malicious tampering with
+Integrity fields are not intended to be a general protection against malicious tampering with
 HTTP messages. This can be achieved by combining it with other approaches such
 as transport-layer security or digital signatures.
 
 ## End-to-End Integrity
 
-Representation-Digest and Content-Digest can help detect `representation data` or content modification due to implementation errors,
+Integrity fields can help detect  *representation data* or content modification due to implementation errors,
 undesired "transforming proxies" (see {{Section 7.7 of SEMANTICS}})
 or other actions as the data passes across multiple hops or system boundaries.
-Even a simple mechanism for end-to-end `representation data` integrity is valuable
-because user-agent can validate that resource retrieval succeeded before handing off to a
+Even a simple mechanism for end-to-end  *representation data* integrity is valuable
+because a *user agent* can validate that resource retrieval succeeded before handing off to a
```

Comment:
The grammer fix is good but there doesn't seem to be any need to treat user agent as special by adding '*'